high-level-caiman/src/lower/lower_schedule.rs

```diff
@@ -521,17 +504,26 @@ fn lower_block(funclet: &Funclet<'_>) -> asm::Funclet {
         commands.append(&mut new_cmds);
     }
     commands.extend(lower_terminator(funclet.terminator(), temp_id, funclet));
+    let implicit_default = asm::Tag {
+        quot: Some(asm::RemoteNodeId {
+            funclet: Some(MetaId(META_TIMELINE.to_string())),
+            node: None,
+        }),
+        flow: ir::Flow::Usable,
+    };
     asm::Funclet {
         kind: ir::FuncletKind::ScheduleExplicit,
         header: asm::FuncletHeader {
             name: asm::FuncletId(funclet.name()),
             args: funclet.inputs(),
             ret: funclet.outputs(),
             binding: asm::FuncletBinding::ScheduleBinding(asm::ScheduleBinding {
-                implicit_tags: None,
-                value: Some(funclet.specs().value.clone()),
-                timeline: Some(funclet.specs().timeline.clone()),
-                spatial: Some(funclet.specs().spatial.clone()),
+                implicit_tags: (implicit_default.clone(), implicit_default),
+                meta_map: asm::MetaMapping {
+                    value: (MetaId(META_VALUE.to_string()), funclet.specs().value.clone()),
```

Comment:
Should these all be mapping to the value spec?

Reply:
oops, good catch!  That's probably causing one of the bugs that showed up, thanks!